modules/backend/constants/queryOptions.ts

```diff
@@ -30,3 +30,11 @@ export const getOffencesOverview = () =>
     queryFn: () => clientApi.scanControllerOffenceOverview(),
     select: (res) => res.data.offences,
   })
+
+export const getLicencePlateTicketsAndPermitsInfo = (ecv?: string) =>
+  queryOptions({
+    queryKey: ['licencePlateTicketsPermitsInfo', ecv],
+    queryFn: () => clientApi.scanControllerTicketsAndPermits(ecv!),
```

Comment:
If I understand this correctly, you are telling typescript in the signature of the function that the `ecv` can be `null` or `undefined`. But then you are forcing there with a bang (!) that it cannot be `null`? Why?  
Would it be better to write it in a way that we don't have to do this kind of weird mental gymnastics here? Or am I missing something?